src/logic/Utils.ts

```diff
@@ -242,19 +241,12 @@ export const injectColourScheme = (scheme: ColourScheme | null) => {
     let injectStr = ":root {";
     const themeIniKeys = Object.keys(scheme);
     themeIniKeys.forEach((key) => {
-      injectStr += `--spice-${key}: #${scheme[key]} !important;`;
-      injectStr += `--spice-rgb-${key}: ${hexToRGB(scheme[key])} !important;`;
+      injectStr += `--spice-${key}: #${scheme[key]};`;
+      injectStr += `--spice-rgb-${key}: ${hexToRGB(scheme[key])};`;
     });
     injectStr += "}";
     schemeTag.innerHTML = injectStr;
-    document.head.appendChild(schemeTag);
```

Comment:
Why is this stuff removed again?

Reply:
It is removed
![image](https://user-images.githubusercontent.com/9348108/216109062-09bb63d7-5a3c-4b60-bc5c-632bce371d2f.png)

Reply:
I believe he meant whole `else` statement as well

Reply:
> I believe he meant whole `else` statement as well

He just pinged that single line
![image](https://user-images.githubusercontent.com/75513645/216114299-0892ca19-c560-4e17-a2e1-89f048b9373f.png)

Reply:
I meant all the red stuff in the `else` block. I just did a single line because I was on my phone. I didn't realise we weren't removing the default `colors.css` anymore. I think it's fine then, my bad.

Reply:
On your phone? you could have added a line stating `else` block. I would have explained clearly lol

Reply:
I thought the "why was this removed" implied the stuff that was removed. ¯\_(ツ)_/¯